Features/Stories/Fieldtypes/EZP-2477-AuthorsFieldType.feature

```diff
@@ -0,0 +1,97 @@
+Feature: Test the validations done on Author fieldtype from Editorial Interface
+
+    @javascript
+    Scenario: Message when using empty author field
+        Given I go to PlatformUI app with username "admin" and password "publish"
```

Comment:
All the administrator logins could be replaced by sentences such:
        Given I am logged in as an administrator

And the implementation would login as admin/publish.
Option: We could have a before-scenario hook where the login is made. Either that or including this line would fit the purpose o the login.